src/layer/layer_settings_manager.cpp

```diff
@@ -150,17 +150,17 @@ std::filesystem::path LayerSettings::FindSettingsFile() {
         LSTATUS err = RegOpenKeyEx(hives[hive_index], TEXT("Software\\Khronos\\Vulkan\\Settings"), 0, KEY_READ, &key);
         if (err == ERROR_SUCCESS) {
             TCHAR name[2048];
-            DWORD i = 0, name_size, type, pValues, value_size;
+            DWORD i = 0, name_size, type, p_values, value_size;
```

Comment:
I have never seen this convention before.

Reply:
I grabbed the clang-tidy file from the vulkan validation layers.

Reply:
I have no preference tbh. Just a matter of changing the clang-tidy file.

```
- { key: readability-identifier-naming.VariableCase, value: lower_case }
```